Classes/Controller/NotificationSettingsController.php

```diff
@@ -57,8 +57,8 @@ private function subscriptionsAction(ServerRequestInterface $request): string
                 $selectedUser = $bodySelectedUser;
             }
         }
-        $postedData = GeneralUtility::_POST('tx_noti');
-        if (is_array($postedData)) {
+        $postedData = $parsedBody['tx_noti'] ?? [];
```

Comment:
`$parsedBody` only is set inside the `if ($this->getBackendUser()->isAdmin()) {` block. So it's not always available here.

can you pull the `$parsedBody = $request->getParsedBody();` before the if statement?